src/GlobalStyle.js

```diff
@@ -0,0 +1,12 @@
+import { createGlobalStyle } from 'styled-components';
+
+export default createGlobalStyle`
+  * {
```

Comment:
Not setting up a default font-size? Could be good, then you can use rem throughout your project because it is set here. Same question with padding and margin, I can see you set the margin to 0 in body, but you don't want to reset both of them for all elements? (might save some work later on)

Reply:
Good point, will change that! 👍🏻